src/framework/mlt.vers

```diff
@@ -576,3 +576,21 @@ MLT_6.22.0 {
     mlt_audio_channel_layout_channels;
     mlt_audio_channel_layout_default;
 } MLT_6.20.0;
+
+MLT_6.24.0 {
```

Comment:
MLT version 6.24.0 was released without this. So, we need to change this symbol version to the next release 6.26.0

Reply:
OK. We will change it to 7.0.0 when the time comes.

Reply:
I plan to merge this soon after the next small changes into a v7 development branch. So, let's go ahead and change the symbol version to MLT_7.0.0.

Reply:
done